muvie/data/users.json

```diff
@@ -0,0 +1,22 @@
+{
+    "users": [
```

Comment:
Why users? It is a list of movies right?

Reply:
yes